### DISCUSSION TITLE
Latex format link

### DISCUSSION THREAD

**Discussion by fabricelallemand15** [quarto-dev/quarto-cli#4685] [Q&A]
Hi

I use Quarto pre-release for multi format publishing. I get links to pdf version of my web pages, but I would also be able to get a link to the tex file generated by quarto with the option `keep-tex: true`.

I tried to write `format-links: [pdf, tex]`, but it doesn't work.

Is there a way to do that automatically?

Thanks

**Comment by dragonstyle** [comment]
Two thing to make this work:

1) list `latex` in the list of formats

2) Manage the `pdf` `latex` interaction that we currently have (`pdf` rendering will clean up the latex file, something we'd like to improve).

Two options for you:

```markdown
---
title: "sample"
format:
  html: default
  pdf:
    keep-tex: true
  latex: default
---
```

or 

```markdown
---
title: "test_html"
format:
  html: default
  pdf: default
  latex:
    output-file: "mydoc.tex"
---
```

Either of these should do it.

**Comment by fabricelallemand15** [comment]
Thanks, it works with `format-links: [pdf, latex]` instead of `format-links: [pdf, tex]`...
